rocky/reports/views/mixins.py

```diff
@@ -78,9 +80,6 @@ def save_report(self, report_names: list) -> ReportOOI:
                         if ooi_name in default_name and report_type_name in default_name:
```

Comment:
When this if does not trigger. name stays untouched, but is still used later on at line 93 leading to a unassigned var

Reply:
Could you check the update code, if this still applies? I think I addressed it properly now.

Reply:
I think so, as the unit-test does not fail anymore. But the problem leading up to this is that we end up trying to create a named report where we don't have enough info. Your current fix leaves the name an empty-string. which is still un-useful and unwanted.

Reply:
True, but imo not something created in this PR and it's better this way than it was before, so I'd like to see this merged asap :)

Reply:
On second thought: ooi_name and report_type_name are always present in default_name, since we create default_name programmatically using both ooi_name and report_type_name. We only use it here to check against to determine if we're at the right instance in the list, to update with "updated_name". When the user didn't actually put in a new name "update_name" will be the same as "default_name" and therefor "name_to_save" will never be empty in practise. Is it neat? Not really. Will it break? I don't believe it will.